test/parallel/test-runner-exit-code.js

```diff
@@ -1,13 +1,31 @@
-// https://github.com/nodejs/node/blob/2fd4c013c221653da2a7921d08fe1aa96aaba504/test/parallel/test-runner-exit-code.js
-
+// https://github.com/nodejs/node/blob/26e27424ad91c60a44d3d4c58b62a39b555ba75d/test/parallel/test-runner-exit-code.js
 'use strict'
-
 const common = require('../common')
 const fixtures = require('../common/fixtures')
 const assert = require('assert')
-const { spawnSync } = require('child_process')
-const { promisify } = require('util')
-const setTimeout = promisify(require('timers').setTimeout)
+const { spawnSync, spawn } = require('child_process')
+const { once } = require('events')
+const finished = require('util').promisify(require('stream').finished)
+
+async function runAndKill (file) {
+  if (common.isWindows) {
+    common.printSkipMessage(`signals are not supported in windows, skipping ${file}`)
+    return
+  }
+  let stdout = ''
+  const child = spawn(process.execPath, ['--test', file])
```

Comment:
How does this pass in node 16/14?

Reply:
https://github.com/nodejs/node-core-test/blob/40710523a76bf6cc182120e82763ddbdb3ceb1be/test/common/index.js#L9